index.html

```diff
@@ -9,44 +9,216 @@
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
```

Comment:
Kindly add the missing image to this card

![image (19)](https://github.com/RolandAmini/portfolio-mobile/assets/59707859/a146ab3f-33e4-41b7-8cdd-c4a7d3c5452f)